app/controllers/api/v1/albums_controller.rb

```diff
@@ -11,12 +14,14 @@ def show
   end
 
   def create
-    album = Album.new(album_params)
-    album.user = current_user
-    if album.save
-      render json: { album: album }
-    else
-      render json: { error: album.errors.full_messages }, status: :unprocessable_entity
+    if user_signed_in?
```

Comment:
since we are authorizing the user before this create action, we shouldn't need to check if the user is signed in